examples/simple-mcp/weather.py

```
@@ -0,0 +1,111 @@
+def main() -> int:
+    app = Server("weather")
+
+    @app.list_resources()
+    async def list_resources() -> list[types.Resource]:
+        return [
+            types.Resource(
+                uri=FileUrl(f"file:///{name}.txt"),
+                name=name,
+                mimeType="text/plain",
+            )
+            for name in CITIES.keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)